website/public/js/static.js

```diff
@@ -40,3 +40,20 @@ window.habitrpg = angular.module('habitrpg', ['chieffancypants.loadingBar', 'ui.
     });
   });
 }])
+
+.controller('AccordionCtrl', function() {
```

Comment:
Oh, probably so! I hadn't thought of it because to date I've only played with directives for building `E`lements, but this would be a perfect spot for an `A`ttribute directive.